local/e2e/compose/start_stop_test.go

```diff
@@ -44,55 +44,52 @@ func TestStartStop(t *testing.T) {
 	}
 
 	t.Run("Up a project", func(t *testing.T) {
-		res := c.RunDockerOrExitError("compose", "-f", "./fixtures/start-stop/compose.yml", "--project-name", projectName, "up", "-d")
-		assert.Assert(t, strings.Contains("Container e2e-start-stop_simple_1  Started", res.Stdout()), res.Stdout())
+		res := c.RunDockerCmd("compose", "-f", "./fixtures/start-stop/compose.yml", "--project-name", projectName, "up", "-d")
```

Comment:
Any special reason for removing the exit code check?

Reply:
`RunDockerCmd` [asserts that the exit code is 0](https://github.com/docker/compose-cli/blob/c11a458666ac7e2de05719c5a0f461d0269f7d5d/utils/e2e/framework.go#L218)